Can't run maven's verify phase with success

## Expected Behavior

<!-- Briefly describe what you expect to happen -->
command exits with success.

## Actual Behavior
command exits with error code due to spotbugs report. 

```
[INFO] --- spotbugs:4.1.4:check (validate) @ dapr-sdk-examples ---
[INFO] BugInstance size is 36
[INFO] Error size is 0
[INFO] Total bugs: 36
[ERROR] Medium: Useless control flow in io.dapr.examples.DaprExamplesProtos$SayRequest$Builder.maybeForceBuilderInitialization() [io.dapr.examples.DaprExamplesProtos$SayRequest$Builder] At DaprExamplesProtos.java:[line 341] UCF_USELESS_CONTROL_FLOW
[ERROR] Medium: Useless control flow in io.dapr.examples.DaprExamplesProtos$SayResponse$Builder.maybeForceBuilderInitialization() [io.dapr.examples.DaprExamplesProtos$SayResponse$Builder] At DaprExamplesProtos.java:[line 909] UCF_USELESS_CONTROL_FLOW
[ERROR] High: carrier must be non-null but is marked as nullable [io.dapr.examples.OpenTelemetryInterceptor$1] At OpenTelemetryInterceptor.java:[line 46] NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE
[ERROR] Medium: Possible null pointer dereference in io.dapr.examples.actors.DemoActorClient.callActorForever(int, String, DemoActor) due to return value of called method [io.dapr.examples.actors.DemoActorClient, io.dapr.examples.actors.DemoActorClient] Dereferenced at DemoActorClient.java:[line 81]Known null at DemoActorClient.java:[line 81] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
[ERROR] Medium: Call to method of static java.text.DateFormat in io.dapr.examples.actors.DemoActorImpl.clock(String) [io.dapr.examples.actors.DemoActorImpl] At DemoActorImpl.java:[line 128] STCAL_INVOKE_ON_STATIC_DATE_FORMAT_INSTANCE
[ERROR] Medium: Call to method of static java.text.DateFormat in io.dapr.examples.actors.DemoActorImpl.lambda$receiveReminder$3(String, Integer) [io.dapr.examples.actors.DemoActorImpl] At DemoActorImpl.java:[line 157] STCAL_INVOKE_ON_STATIC_DATE_FORMAT_INSTANCE
[ERROR] Medium: Call to method of static java.text.DateFormat in io.dapr.examples.actors.DemoActorImpl.say(String) [io.dapr.examples.actors.DemoActorImpl] At DemoActorImpl.java:[line 83] STCAL_INVOKE_ON_STATIC_DATE_FORMAT_INSTANCE
[ERROR] High: Found reliance on default encoding in io.dapr.examples.bindings.http.InputBindingController.lambda$handleInputBinding$0(byte[]): new String(byte[]) [io.dapr.examples.bindings.http.InputBindingController] At InputBindingController.java:[line 35] DM_DEFAULT_ENCODING
[ERROR] Medium: Possible null pointer dereference in io.dapr.examples.configuration.grpc.ConfigurationClient.getConfigurations(DaprClient) due to return value of called method [io.dapr.examples.configuration.grpc.ConfigurationClient, io.dapr.examples.configuration.grpc.ConfigurationClient] Dereferenced at ConfigurationClient.java:[line 69]Known null at ConfigurationClient.java:[line 69] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
[ERROR] Medium: Nullcheck of client at line 51 of value previously dereferenced in io.dapr.examples.configuration.grpc.ConfigurationClient.main(String[]) [io.dapr.examples.configuration.grpc.ConfigurationClient, io.dapr.examples.configuration.grpc.ConfigurationClient] At ConfigurationClient.java:[line 51]Redundant null check at ConfigurationClient.java:[line 52] RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE
[ERROR] Medium: Possible null pointer dereference in io.dapr.examples.configuration.http.ConfigurationClient.getConfigurations(DaprClient) due to return value of called method [io.dapr.examples.configuration.http.ConfigurationClient, io.dapr.examples.configuration.http.ConfigurationClient] Dereferenced at ConfigurationClient.java:[line 72]Known null at ConfigurationClient.java:[line 72] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
[ERROR] Medium: Nullcheck of client at line 49 of value previously dereferenced in io.dapr.examples.configuration.http.ConfigurationClient.main(String[]) [io.dapr.examples.configuration.http.ConfigurationClient, io.dapr.examples.configuration.http.ConfigurationClient] At ConfigurationClient.java:[line 49]Redundant null check at ConfigurationClient.java:[line 50] RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE
[ERROR] Medium: Call to method of static java.text.DateFormat in io.dapr.examples.invoke.grpc.HelloWorldService$GrpcHelloWorldDaprService.say(DaprExamplesProtos$SayRequest) [io.dapr.examples.invoke.grpc.HelloWorldService$GrpcHelloWorldDaprService] At HelloWorldService.java:[line 129] STCAL_INVOKE_ON_STATIC_DATE_FORMAT_INSTANCE
[ERROR] Medium: Format string should use %n rather than \n in io.dapr.examples.invoke.grpc.HelloWorldService$GrpcHelloWorldDaprService.start(int) [io.dapr.examples.invoke.grpc.HelloWorldService$GrpcHelloWorldDaprService] At HelloWorldService.java:[line 73] VA_FORMAT_STRING_USES_NEWLINE
[ERROR] Medium: Call to method of static java.text.DateFormat in io.dapr.examples.invoke.http.DemoServiceController.lambda$handleMethod$0(byte[], Map) [io.dapr.examples.invoke.http.DemoServiceController] At DemoServiceController.java:[line 60] STCAL_INVOKE_ON_STATIC_DATE_FORMAT_INSTANCE
[ERROR] High: Found reliance on default encoding in io.dapr.examples.invoke.http.InvokeClient.main(String[]): new String(byte[]) [io.dapr.examples.invoke.http.InvokeClient] At InvokeClient.java:[line 45] DM_DEFAULT_ENCODING
[ERROR] Medium: Possible null pointer dereference in io.dapr.examples.invoke.http.InvokeClient.main(String[]) due to return value of called method [io.dapr.examples.invoke.http.InvokeClient, io.dapr.examples.invoke.http.InvokeClient] Method invoked at InvokeClient.java:[line 45]Known null at InvokeClient.java:[line 44] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
[ERROR] Medium: Nullcheck of client at line 63 of value previously dereferenced in io.dapr.examples.pubsub.BulkPublisher.main(String[]) [io.dapr.examples.pubsub.BulkPublisher, io.dapr.examples.pubsub.BulkPublisher] At BulkPublisher.java:[line 63]Redundant null check at BulkPublisher.java:[line 98] RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE
[ERROR] Medium: Nullcheck of client at line 77 of value previously dereferenced in io.dapr.examples.pubsub.CloudEventBulkPublisher.main(String[]) [io.dapr.examples.pubsub.CloudEventBulkPublisher, io.dapr.examples.pubsub.CloudEventBulkPublisher] At CloudEventBulkPublisher.java:[line 77]Redundant null check at CloudEventBulkPublisher.java:[line 91] RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE
[ERROR] Medium: Nullcheck of client at line 66 of value previously dereferenced in io.dapr.examples.pubsub.CloudEventPublisher.main(String[]) [io.dapr.examples.pubsub.CloudEventPublisher, io.dapr.examples.pubsub.CloudEventPublisher] At CloudEventPublisher.java:[line 66]Redundant null check at CloudEventPublisher.java:[line 84] RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE
[ERROR] Medium: Nullcheck of client at line 56 of value previously dereferenced in io.dapr.examples.pubsub.Publisher.main(String[]) [io.dapr.examples.pubsub.Publisher, io.dapr.examples.pubsub.Publisher] At Publisher.java:[line 56]Redundant null check at Publisher.java:[line 76] RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE
[ERROR] Medium: Nullcheck of client at line 63 of value previously dereferenced in io.dapr.examples.pubsub.PublisherWithTracing.main(String[]) [io.dapr.examples.pubsub.PublisherWithTracing, io.dapr.examples.pubsub.PublisherWithTracing] At PublisherWithTracing.java:[line 63]Redundant null check at PublisherWithTracing.java:[line 88] RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE
[ERROR] Medium: Format string should use %n rather than \n in io.dapr.examples.pubsub.http.SubscriberController.lambda$handleBulkMessage$3(BulkSubscribeMessage) [io.dapr.examples.pubsub.http.SubscriberController] At SubscriberController.java:[line 123] VA_FORMAT_STRING_USES_NEWLINE
[ERROR] Medium: Format string should use %n rather than \n in io.dapr.examples.pubsub.http.SubscriberController.lambda$handleBulkMessage$3(BulkSubscribeMessage) [io.dapr.examples.pubsub.http.SubscriberController] At SubscriberController.java:[line 121] VA_FORMAT_STRING_USES_NEWLINE
[ERROR] Medium: Possible null pointer dereference in io.dapr.examples.querystate.QuerySavedState.main(String[]) due to return value of called method [io.dapr.examples.querystate.QuerySavedState, io.dapr.examples.querystate.QuerySavedState] Dereferenced at QuerySavedState.java:[line 102]Known null at QuerySavedState.java:[line 100] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
[ERROR] Medium: Possible null pointer dereference in io.dapr.examples.state.StateClient.main(String[]) due to return value of called method [io.dapr.examples.state.StateClient, io.dapr.examples.state.StateClient] Dereferenced at StateClient.java:[line 74]Known null at StateClient.java:[line 74] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
[ERROR] Medium: Possible null pointer dereference in io.dapr.examples.state.StateClient.main(String[]) due to return value of called method [io.dapr.examples.state.StateClient, io.dapr.examples.state.StateClient] Dereferenced at StateClient.java:[line 93]Known null at StateClient.java:[line 93] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
[ERROR] Medium: Possible null pointer dereference in io.dapr.examples.state.StateClient.main(String[]) due to return value of called method [io.dapr.examples.state.StateClient, io.dapr.examples.state.StateClient] Dereferenced at StateClient.java:[line 112]Known null at StateClient.java:[line 112] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
[ERROR] Medium: Possible null pointer dereference in io.dapr.examples.state.StateClient.main(String[]) due to return value of called method [io.dapr.examples.state.StateClient, io.dapr.examples.state.StateClient] Dereferenced at StateClient.java:[line 123]Known null at StateClient.java:[line 123] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
[ERROR] Medium: Nullcheck of client at line 60 of value previously dereferenced in io.dapr.examples.state.StateClient.main(String[]) [io.dapr.examples.state.StateClient, io.dapr.examples.state.StateClient] At StateClient.java:[line 60]Redundant null check at StateClient.java:[line 128] RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE
[ERROR] High: Found reliance on default encoding in io.dapr.examples.tracing.InvokeClient.lambda$main$0(byte[]): new String(byte[]) [io.dapr.examples.tracing.InvokeClient] At InvokeClient.java:[line 63] DM_DEFAULT_ENCODING
[ERROR] Medium: Call to method of static java.text.DateFormat in io.dapr.examples.tracing.TracingDemoServiceController.lambda$handleMethod$0(String, Map) [io.dapr.examples.tracing.TracingDemoServiceController] At TracingDemoServiceController.java:[line 63] STCAL_INVOKE_ON_STATIC_DATE_FORMAT_INSTANCE
[ERROR] Medium: Possible null pointer dereference in io.dapr.examples.tracing.Validation.doValidate() due to return value of called method [io.dapr.examples.tracing.Validation, io.dapr.examples.tracing.Validation] Dereferenced at Validation.java:[line 82]Known null at Validation.java:[line 82] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
[ERROR] Medium: Possible null pointer dereference in io.dapr.examples.unittesting.DaprExampleTest$MyApp.getState() due to return value of called method [io.dapr.examples.unittesting.DaprExampleTest$MyApp, io.dapr.examples.unittesting.DaprExampleTest$MyApp] Dereferenced at DaprExampleTest.java:[line 72]Known null at DaprExampleTest.java:[line 72] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
[ERROR] Medium: Should io.dapr.examples.unittesting.DaprWorkflowExampleTest$DemoWorkflow be a _static_ inner class? [io.dapr.examples.unittesting.DaprWorkflowExampleTest$DemoWorkflow] At DaprWorkflowExampleTest.java:[lines 45-59] SIC_INNER_SHOULD_BE_STATIC
[ERROR] Medium: Nullcheck of runtime at line 37 of value previously dereferenced in io.dapr.examples.workflows.DemoWorkflowWorker.main(String[]) [io.dapr.examples.workflows.DemoWorkflowWorker, io.dapr.examples.workflows.DemoWorkflowWorker] At DemoWorkflowWorker.java:[line 37]Redundant null check at DemoWorkflowWorker.java:[line 38] RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE
[INFO] 
``` 


## Steps to Reproduce the Problem

Run `mvn verify` 

## Release Note

<!-- How should the fix for this issue be communicated in our release notes? It can be populated later. -->
<!-- Keep it as a single line. Examples: -->

<!-- RELEASE NOTE: **ADD** New feature in Dapr. -->
<!-- RELEASE NOTE: **FIX** Bug in runtime. -->
<!-- RELEASE NOTE: **UPDATE** Runtime dependency. -->

RELEASE NOTE:

--- a/pom.xml
+++ b/pom.xml
@@ -29,6 +29,7 @@
     <gpg.skip>true</gpg.skip>
     <spotbugs.version>4.0.0-RC1</spotbugs.version>
     <spotbugs.fail>true</spotbugs.fail>
+    <spotbugs.exclude.filter.file>../spotbugs-exclude.xml</spotbugs.exclude.filter.file>
     <springboot.version>2.7.8</springboot.version>
     <argLine>--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED</argLine>
   </properties>
@@ -219,6 +220,7 @@
           </dependency>
         </dependencies>
         <configuration>
+          <excludeFilterFile>${spotbugs.exclude.filter.file}</excludeFilterFile>
           <failOnError>${spotbugs.fail}</failOnError>
           <xmlOutput>true</xmlOutput>
         </configuration>

--- a/spotbugs-exclude.xml
+++ b/spotbugs-exclude.xml
@@ -1,0 +1,6 @@
+<FindBugsFilter>
+    <!--Ignoring checking for examples-->
+    <Match>
+        <Package name="~io\.dapr\.examples.*"/>
+    </Match>
+</FindBugsFilter>

